drivers/syslog/syslog.h

```diff
@@ -72,17 +87,19 @@ EXTERN FAR const struct syslog_channel_s *g_syslog_channel
  *   SYSLOG device.  That would be a good extension.
  *
  * Input Parameters:
- *   devpath - The full path to the character device to be used.
- *   oflags  - File open flags
- *   mode    - File open mode (only if oflags include O_CREAT)
+ *   syslog_dev - Handle to syslog device to be used.
+ *   devpath    - The full path to the character device to be used.
+ *   oflags     - File open flags
+ *   mode       - File open mode (only if oflags include O_CREAT)
```

Comment:
```suggestion
 *   oflags     - File open flags.
 *   mode       - File open mode (only if oflags include O_CREAT).
```
nit: Just for consistency.

Reply:
Done.